resources/views/site/feedback.blade.php

```diff
@@ -67,6 +67,8 @@
         <div class="g-recaptcha" data-sitekey="6LcC3XoUAAAAAG8ST6HXqS3_reIZRLcA09sDdodw"></div>
         <br>
         <button class="btn btn-success" type="submit">Send Feedback</button>
+        <br>
+        <br>
```

Comment:
I think we can add a class to the button -> `<button class="btn btn-success mb-2" ...` that will do the same thing and is a little cleaner

Reply:
Yeah, you're right! Fixed.